resources/themes/pterodactyl/admin/nodes/view/allocation.blade.php

```diff
@@ -159,7 +159,7 @@
                     </div>
                 </div>
                 <div class="modal-footer">
-                    {{{ csrf_field() }}}
+                    {{ csrf_field() }}
```

Comment:
Use csrf blade helper?

Reply:
you caught me before i did the bulk change, this was just to fix the unnecessary triple curly brace